src/components/LogIn.js

```diff
@@ -22,18 +23,24 @@ const LogIn = ({ authMemberDoc, noRedirect }) => {
       }
     },
   };
-  if (authMemberDoc && !noRedirect) {
-    return (
-      <Redirect to={`/m/${authMemberDoc.get('mid')}`} replace />
```

Comment:
@rahulgi is it okay for me to take away the `replace`? Could not find any documentation for it

Reply:
Yep, that's okay. I think I might've been mistaken in putting it there - it appears it's a property on `Link`s but not on `Redirects` as `replace` is the default behavior for those anyway.

Reply:
More reason for TypeScript! Eliminate possibility for unintended, long-standing misuse of libraries; and remove confusion and unnecessary communication over errant attributes like this.